packages/psss/src/utils/auth.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { AccessPolicy } from '@tupaia/access-policy';
+
+export const checkIsAuthorisedForMultiCountry = user => {
+  const activeEntity = getActiveEntityByUser(user);
+  return Array.isArray(activeEntity);
+};
+
+export const checkIsAuthorisedForCountry = (user, match) => {
+  if (checkIsAuthorisedForMultiCountry(user)) {
+    return true;
+  }
+
+  return getActiveEntityByUser(user) === match.params.countryCode;
+};
+
+export const getActiveEntityByUser = user => {
+  if (!user) {
+    return null;
+  }
+
+  const accessPolicy = new AccessPolicy(user.accessPolicy);
+  // Todo: Update with the correct access policy check
+  // @see: https://github.com/beyondessential/tupaia-backlog/issues/1268
+
+  // To Test switch between Admin and Public
+  const psssPermissions = accessPolicy.allows('DL', 'Public');
+  if (psssPermissions) {
+    return ['as', 'to'];
+  }
+  // Todo: Return the correct activeCountry
+  // return as lower case for consistency
+  return 'as';
```

Comment:
It might make things more predictable to always return an array, and then have the selectors that check if it's a multi country user check whether that array is length one?

Reply:
Yes that makes more sense. Initially I thought the permissions were regional or single country but things make more sense now that it is just a list of countries that can be one or more!